spyder/app/mainwindow.py

```diff
@@ -3505,11 +3509,19 @@ def _dialog_finished(result_code):
                 if widget is not None:
                     dlg.add_page(widget)
 
-            for plugin in [self.appearance, self.run, self.workingdirectory,
-                           self.editor, self.projects, self.ipyconsole,
-                           self.historylog, self.help, self.variableexplorer,
-                           self.onlinehelp, self.explorer, self.findinfiles
-                           ] + self.thirdparty_plugins:
+            for plugin in [self.appearance,
+                           self.run,
+                           self.shortcuts,
+                           self.workingdirectory,
+                           self.editor,
+                           self.projects,
+                           self.ipyconsole,
+                           self.historylog,
+                           self.help,
+                           self.variableexplorer,
+                           self.onlinehelp,
+                           self.explorer,
+                           self.findinfiles] + self.thirdparty_plugins:

```

Comment:
Remaining PRs will keep generating conflicts. So I made the list like this to avoid that.

Also, this list will be gone once all is migrated since Plugins will be loaded from entry points instead of hardcoded.